assets/sass/patterns/molecules/aside.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../definitions";
+
+.aside {
```

Comment:
Rename to `content-aside`.

Reply:
Class and name file have been refactored to 'content-aside'